packages/amplify-category-function/src/provider-utils/awscloudformation/index.ts

```diff
@@ -119,21 +119,23 @@ export async function addFunctionResource(
     await openEditor(context, category, completeParams.resourceName, completeParams.functionTemplate);
   }
 
-  const { print } = context;
+  if (!completeParams.skipNextSteps) {
+    const { print } = context;
 
-  const customPoliciesPath = pathManager.getCustomPoliciesPath(category, completeParams.resourceName);
+    const customPoliciesPath = pathManager.getCustomPoliciesPath(category, completeParams.resourceName);
 
-  print.success(`Successfully added resource ${completeParams.resourceName} locally.`);
-  print.info('');
-  print.success('Next steps:');
-  print.info(`Check out sample function code generated in <project-dir>/amplify/backend/function/${completeParams.resourceName}/src`);
-  print.info('"amplify function build" builds all of your functions currently in the project');
-  print.info('"amplify mock function <functionName>" runs your function locally');
-  print.info(`To access AWS resources outside of this Amplify app, edit the ${customPoliciesPath}`);
-  print.info('"amplify push" builds all of your local backend resources and provisions them in the cloud');
-  print.info(
-    '"amplify publish" builds all of your local backend and front-end resources (if you added hosting category) and provisions them in the cloud',
-  );
+    print.success(`Successfully added resource ${completeParams.resourceName} locally.`);
```

Comment:
nit: would be nice to switch to `amplify-prompts` for these print statements

Reply:
I've few more fast follow tasks related to this module, I will include this.